ui-components/src/page_components/listing/listing_sidebar/events/DownloadLotteryResults.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react"
 import { ListingEvent, ListingEventType } from "@bloom-housing/backend-core/types"
 import { t } from "../../../../helpers/translator"
-import { pdfUrlFromListingEvents } from "../../../../helpers/pdfs"
+import { pdfUrlFromListingEvents } from "@bloom-housing/shared-helpers"
```

Comment:
Same deal here…

Reply:
Refactored this component to remove the dependency, ty!